src/identity.ts

```diff
@@ -167,7 +167,7 @@ export const deleteIdentity = async (serviceId: string, identityId: string): Pro
   const identity = await superagent
     .get(`${baseUrl}${conUrl}/identity/${identityId}`)
     .set('User-Agent', EveryAuthVersion)
-    .set('Authorization', `Bearer ${profile.token}`)
+    .set('Authorization', `Bearer ${profile.accessToken}`)
```

Comment:
Why has this changed? Isn't this the same profile format as before?

Reply:
I normalized everything onto `accessToken` instead of using `token` some places and `accessToken` other places.  A largely internal change, driven by the output of `everyauth token`.

Reply:
Maybe I am missing something about this change, but in my mind the code should remain unchanged when people graduate from `everyauth` to `fuse` CLI. Is this the case?

Reply:
Yes.